relay-system/src/controller.rs

```diff
@@ -55,11 +58,22 @@ pub use actix::actors::signal::{Signal, SignalType};
 ///     Ok(())
 /// }).unwrap();
 /// ```
+
+type ShutdownReceiver = watch::Receiver<Option<Shutdown>>; // Find a good name for this
+type ShutdownSender = watch::Sender<Option<Shutdown>>; // Check of that is really needed
+                                                       // Only used once but gives a nice symetry
+
+/// TODO
```

Comment:
If you'd like to write a nice doc comment for the controller go ahead, but also this isn't a new struct so you could skip that if you like.